htdocs/api/class/api_documents.class.php

```diff
@@ -472,6 +472,20 @@ public function getDocumentsListByElement($modulepart, $id = 0, $ref = '', $sort
 			}
 
 			$upload_dir = $conf->knowledgemanagement->dir_output.'/knowledgerecord/'.dol_sanitizeFileName($object->ref);
+		} elseif ($modulepart == 'ticket') {
+			require_once DOL_DOCUMENT_ROOT.'/ticket/class/ticket.class.php';
+
+			if (!DolibarrApiAccess::$user->rights->societe->client->voir && !DolibarrApiAccess::$user->rights->societe->client->voir) {
```

Comment:
Permission seems the permission of another module.

Reply:
into the ticket class all the rights are of that kind : $user->rights->societe->client->voir

Reply:
No the use of $user->rights->societe->client->voir is only when we need a link with a sale representative of a thirdparty.
For ticket CRUD method, permissions must be the permission of module ticket so user->rights->ticket->...

Reply:
I made the rights changes